packages/back-end/test/api/api.setup.ts

```diff
@@ -9,7 +9,7 @@ import { getAgendaInstance } from "back-end/src/services/queueing";
 
 jest.mock("back-end/src/util/secrets", () => ({
   ...jest.requireActual("back-end/src/util/secrets"),
-  CRON_ENABLED: 1,
+  CRON_ENABLED: 0,
```

Comment:
I'm not sure why we would have wanted CRON_ENABLED while running api tests.  Some of the jobs were throwing errors during the test runs.